compass/ocean/tests/dam_break/lts/lts_regions.py

```diff
@@ -0,0 +1,328 @@
+
+                                lts_rgn[cell1] = 4
+                                changed_cells[(i + 1) % 2].append(cell1)
+                        else:
+                            if (lts_rgn[cell1] == 4 and lts_rgn[cell2] == 2):
+
+                                lts_rgn[cell2] = 4
+                                changed_cells[(i + 1) % 2].append(cell2)
+
+                            elif (lts_rgn[cell1] == 2 and lts_rgn[cell2] == 4):
+
+                                lts_rgn[cell1] = 4
+                                changed_cells[(i + 1) % 2].append(cell1)
+
+    # create lts_mesh.nc
+
+    logger.info('Creating lts_mesh...')
+
+    # open mesh nc file to be copied
+
+    ds_msh = xr.open_dataset(mesh)
+    ds_ltsmsh = ds_msh.copy(deep=True)
+    ltsmsh_name = 'lts_mesh.nc'
+    write_netcdf(ds_ltsmsh, ltsmsh_name)
+    mshnc = nc.Dataset(ltsmsh_name, 'a', format='NETCDF4_64BIT_OFFSET')
+
+    try:
+        # try to get LTSRegion and assign new value
+        lts_rgn_NC = mshnc.variables['LTSRegion']
+        lts_rgn_NC[:] = lts_rgn[:]
+    except KeyError:
+        # create new variable
+        ncells_NC = mshnc.dimensions['nCells'].name
+        lts_rgn_NC = mshnc.createVariable('LTSRegion', np.int32, (ncells_NC,))
+
+        # set new variable
+        lts_rgn_NC[:] = lts_rgn[:]
+
+    mshnc.close()
+
+    # open init nc file to be copied
+
+    ds_init = xr.open_dataset(init)
+    ds_ltsinit = ds_init.copy(deep=True)
+    ltsinit_name = 'lts_ocean.nc'
+    write_netcdf(ds_ltsinit, ltsinit_name)
+    initnc = nc.Dataset(ltsinit_name, 'a', format='NETCDF4_64BIT_OFFSET')
+
+    try:
+        # try to get LTSRegion and assign new value
+        lts_rgn_NC = initnc.variables['LTSRegion']
+        lts_rgn_NC[:] = lts_rgn[:]
+    except KeyError:
+        # create new variable
+        ncells_NC = initnc.dimensions['nCells'].name
+        lts_rgn_NC = initnc.createVariable('LTSRegion', np.int32, (ncells_NC,))
+
+        # set new variable
+        lts_rgn_NC[:] = lts_rgn[:]
+
+    initnc.close()
+
+    extract_vtk(ignore_time=True, lonlat=0,
+                dimension_list=['maxEdges='],
+                variable_list=['allOnCells'],
+                filename_pattern=ltsmsh_name,
+                out_dir='lts_mesh_vtk', use_progress_bar=use_progress_bar)
+
+    # label cells in graph.info
+
+    logger.info('Weighting ' + graph_info + '...')
+
+    fine_cells = 0
+    coarse_cells = 0
+
+    newf = ""
+    with open(graph_info, 'r') as f:
+        lines = f.readlines()
+        # this is to have fine, coarse and interface be separate for METIS
+        # newf += lines[0].strip() + " 010 3 \n"
+
+        # this is to have fine, and interface be together for METIS
+        newf += lines[0].strip() + " 010 2 \n"
+        for icell in range(1, len(lines)):
+            if (lts_rgn[icell - 1] == 1 or lts_rgn[icell - 1] == 5):  # fine
+
+                # newf+= "0 1 0 " + lines[icell].strip() + "\n"
```

Comment:
I don't think so, why?

Reply:
Oh, just that they can be deleted rather than commented out.

Reply:
Even though I do not anticipate using them in the future, I would rather have them there as a record in case we will need to use that old METIS partitioning for some reason. If I remove them then I will not remember how they were :)

Reply:
Ok. Why don't you leave a comment that mentions that they refer to the old METIS partitioning so others know why they are there?